src/images/minio.rs

```diff
@@ -0,0 +1,88 @@
+use crate::{core::WaitFor, Image, ImageArgs};
+use std::collections::HashMap;
+
+const NAME: &str = "minio/minio";
+const TAG: &str = "latest";
+
+const DIR: &str = "/data";
+const CONSOLE_ADDRESS: &str = ":9001";
+
+#[derive(Debug)]
+pub struct MinIO {
+    env_vars: HashMap<String, String>,
+    tag: String,
+}
+
+impl Default for MinIO {
+    fn default() -> Self {
+        Self::new_with_tag(TAG.to_owned())
+    }
+}
+
+impl MinIO {
+    pub fn new(tag: String, env_vars: HashMap<String, String>) -> Self {
+        Self { tag, env_vars }
+    }
+
+    pub fn new_with_tag(tag: String) -> Self {
```

Comment:
The builder on `RunnableImage` already allows to override this!

Reply:
I think they can coexist 🤔 
The tag is quite an important part and it's easier to specify it in the constructor than to create a wrapper.
```rs
// RunnableImage approach
let image = RunnableImage::from(images::minio::MinIO::default())
        .with_tag("latest");
        
// Direct approach        
let image = images::minio::MinIO::with_tag("latest");

```

If you disagree, let me know, I'll remove the constructors. No problems.

Reply:
Perhaps this distorts the general interface for working with images, but it seems worth documenting `RunnableImage` 🤔

Reply:
You don't need to create the wrapper, there is an extension trait so you can call `with_tag` directly on the image :)